### ISSUE TITLE
@types/braintree-web-drop-in PaypalCreateOptions is not complete

### ISSUE THREAD

**Issue by celiamercier** [DefinitelyTyped/DefinitelyTyped#38771]
I use braintree-web-drop-in with typescript and I noticed that the paypalCreateOptions interface is not complete.

Current definition is the following : 
```typescript
export interface paypalCreateOptions {
    flow: "checkout" | "vault";
    amount?: string | number;
    currency?: string;
    buttonStyle?: string;
    commit?: boolean;
}
```

But some properties are missing like 'locale' for example. Furthermore, the buttonStyle should not be a string but an object.

Documentation for the buttonStyle can be found here : https://developer.paypal.com/docs/archive/checkout/how-to/customize-button/#button-styles

and documentation for all the create options can be found here : 
https://braintree.github.io/braintree-web/3.53.0/PayPalCheckout.html#createPayment

**Comment by iamricard** [comment]
Hi @celiamercier, I believe this was fixed in #49491 - sorry this took so long 😄

**Comment by orta** [comment]
Hi thread, we're moving DefinitelyTyped to use [GitHub Discussions](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/53377) for conversations the `@types` modules in DefinitelyTyped.

To help with the transition, we're closing all issues which haven't had activity in the last 6 months, which includes this issue. If you think closing this issue is a mistake, please pop into the [TypeScript Community Discord](https://discord.gg/typescript) and mention the issue in the `definitely-typed` channel.
